src/javascript/registerCEActions.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import {ArrowBack} from '@material-ui/icons';
 import {SelectorTypes} from '~/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/SelectorTypes';
-import {registerActions as registerEditActions} from '~/Edit';
-import {registerActions as registerCreateActions} from '~/Create';
+import registerEditActions from '~/Edit';
+import registerCreateActions from '~/Create';
```

Comment:
import from ~/Edit/registerEditActions  and ~/Create/registerCreateActions ( or eventually rename to [Edit|Create].actions.js to be consistent with cmm )

Reply:
named exports and same names as CMM is now used here